### ISSUE TITLE
strictyaml permits comma-separated fields, but these are not YAML

### ISSUE THREAD

**Issue by grahamgower** [popsim-consortium/demes-python#131]
E.g.
```yaml
demes:
- id: A
  initial_size: 1000
- id: B
  initial_size: 1000
  ancestors: A
```
Is more correctly expressed as:
```yaml
demes:
- id: A
  initial_size: 1000
- id: B
  initial_size: 1000
  ancestors:
  - A
```
I guess this is fine if we're happy to convert comma-separated fields into lists ourselves. But any other consumers of the YAML files will have to do likewise.

**Comment by jeromekelleher** [comment]
I would argue for sticking with the yaml spec, such as it is. Ironic that strictyaml decided to add its own extensions...

**Comment by apragsdale** [comment]
This was closed by #132, yes?

**Comment by grahamgower** [comment]
#132 didn't change the behaviour. We still use comma-separated fields in the YAML.
